parcels/particleset/particlesetsoa.py

```diff
@@ -273,6 +316,14 @@ def error_particles(self):
         error_indices = self.data_indices('state', [StateCode.Success, StateCode.Evaluate], invert=True)
         return ParticleCollectionIterableSOA(self._collection, subset=error_indices)
 
+    def active_particles_mask(self, time, dt):
+        # TODO: make this into a mask
```

Comment:
So this still needs to be done before PR merging? Or after?

Reply:
It is already done.